lua/relative-toggle/init.lua

```diff
@@ -15,11 +15,15 @@ local function create_autocmd(event, opts)
 end
 
 ---@param relative boolean #whether relativenumber should be set
-local function set_relativenumber(relative)
+---@param redraw boolean #whether to redraw the screen
+local function set_relativenumber(relative, redraw)
     local in_insert_mode = vim.api.nvim_get_mode().mode == "i"
 
     if vim.o.number then
         vim.opt.relativenumber = relative and not in_insert_mode
+        if redraw then
```

Comment:
```suggestion

        if redraw then
```

A new line should be there for better readability